loadtest/control/utils.go

```diff
@@ -276,31 +282,50 @@ func AttachFilesToPost(u user.User, post *model.Post) error {
 // AttachFilesToDraft uploads at least one file on behalf of the user, attaching
 // all uploaded files to the draft.
 func AttachFilesToDraft(u user.User, draft *model.Draft) error {
-	fileIDs, err := _attachFilesToObj(u, draft.ChannelId)
+	fileIDs, err := _attachFilesToObj(u, draft.ChannelId, rand.Intn(4))
 	if err != nil {
 		return err
 	}
 	draft.FileIds = fileIDs
 	return nil
 }
 
-func _attachFilesToObj(u user.User, channelID string) ([]string, error) {
+func AttachFileToBookmark(u user.User, bookmark *model.ChannelBookmark) error {
+	fileIDs, err := _attachFilesToObj(u, bookmark.ChannelId, 1)
+	if err != nil {
+		return err
+	}
+
+	bookmark.FileId = fileIDs[0]
+	return nil
+}
+
+func _attachFilesToObj(u user.User, channelID string, qtyToUpload int) ([]string, error) {
 	type file struct {
 		data   []byte
 		upload bool
 	}
 	filenames := []string{"test_upload.png", "test_upload.jpg", "test_upload.mp4", "test_upload.txt"}
 	files := make(map[string]*file, len(filenames))
 
+	count := 0
+
 	for _, filename := range filenames {
+		upload := rand.Intn(2) == 0 && qtyToUpload > count
+		if upload {
+			count += 1
+		}
```

Comment:
Wondering if it would be simpler to break it into two?

```go
upload := rand.Intn(2) == 0
if upload {
	count += 1
}
files[filename] = &file{
	data:   MustAsset(filename),
	upload: upload,
}
if count == qtyToUpload {
  break
}
```